### ISSUE TITLE
fitgauss = TRUE results in NA with R4.10 and xcms_3.14.0

### ISSUE THREAD

**Issue by pmbrophy** [sneumann/xcms#573]
I am processing individual .mzML files with the xcmsSet() function as follows: 

```r
result <- xcms::xcmsSet(files = "./foobarPath/msFile.mzML",
                        method = "centWave", 
                        noise = 100, 
                        ppm = 25,                 
                        prefilter = c(5, 1000),   
                        snthresh = 10, 
                        peakwidth = c(1.5,5),   
                        mzCenterFun = "wMean", 
                        mzdiff = 0.01,
                        integrate = 2,            
                        lock = FALSE,             
                        fitgauss = TRUE,
                        verbose.columns = TRUE)

peaks <- xcms::peakTable(result)
```
Using R Version 4.0.2 and xcms_3.12.0, peaks contains numeric values in the columns "egauss", "mu", "sigma", and "h". I recently upgraded to R Version 4.1.0 and xcms_3.14.0. Running the same functions on the same files on the same computer(s) (Windows 10) results in "egauss", "mu", "sigma", and "h" containing only "NA" values.

**Comment by jorainer** [comment]
Thanks @pmbrophy for reporting. I can confirm this locally and will look into it.

**Comment by jorainer** [comment]
This has been fixed in `xcms` version 3.14.1. This version will soon be available through Bioconductors update routine (i.e. can be updated with `BiocManager("xcms")`). Until then it can be manually installed with `devtools::install_github("sneumann/xcms", ref = "RELEASE_3_13")`.

Please test the updated version and close this issue if it fixes your problem.

**Comment by pmbrophy** [comment]
@jorainer `xcms` version 3.14.1 solved the issue.
